docs/src/main/asciidoc/keycloak-authorization-guide.adoc

```diff
@@ -170,7 +167,7 @@ quarkus.oidc.credentials.secret=secret
 quarkus.keycloak.policy-enforcer.enable=true
 ----
 
-=== Configuring CORS
+==== Configuring CORS
```

Comment:
I believe the CORS section should not be in this doc - the users should follow the OIDC guide for the general overview of how to set up the flows themselves